backend/src/main/java/dev/k2/deleteXsdFile/DeleteXsdFileService.java

```diff
@@ -0,0 +1,35 @@
+package dev.k2.deleteXsdFile;
+
+import dev.k2.commons.FileDescription;
+import dev.k2.commons.XsdFileDescriptionRepository;
+import dev.k2.commons.XsdFileRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Service;
+
+import java.io.FileNotFoundException;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class DeleteXsdFileService {
+
+    JpaRepository<FileDescription, Long> repository;
```

Comment:
This repository is always null.
![image](https://user-images.githubusercontent.com/32412218/203780865-e3e7d915-791f-43c2-b2a6-49ad843ce3fb.png)